actionpack/lib/action_controller/metal/strong_parameters.rb

```diff
@@ -252,6 +252,10 @@ def require(key)
     # Alias of #require.
     alias :required :require
 
+    def require_all(keys)
```

Comment:
Why not just make `def require(*keys)` and loop through them, instead of adding another mehod.